okhttp/src/jvmTest/java/okhttp3/RouteFailureTest.kt

```diff
@@ -0,0 +1,240 @@
+package okhttp3
+
+
+import java.io.IOException
+import java.net.InetAddress
+import java.net.InetSocketAddress
+import java.net.Socket
+import java.net.SocketAddress
+import mockwebserver3.MockResponse
+import mockwebserver3.MockWebServer
+import mockwebserver3.SocketPolicy
+import mockwebserver3.junit5.internal.MockWebServerInstance
+import okhttp3.internal.http2.ErrorCode
+import okhttp3.testing.Flaky
+import okhttp3.testing.PlatformRule
+import okhttp3.tls.internal.TlsUtil.localhost
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
+
+@Flaky
```

Comment:
eep

Reply:
Is this due to raciness in fast fallback? I wonder if we can add something to make it less racy, like changing the timeouts to MAX_VALUE or 0 or something

Reply:
Sorry, not Flaky. I'm just using it for logging the tracing.